.github/workflows/build.yml

```
@@ -0,0 +1,81 @@
+          token: ${{ secrets.CODECOV_TOKEN }}
+
+      - name: Expo doctor
+        run: bunx expo-doctor
+
+      - name: Build app bundles
+        run: bunx expo export --platform all --output-dir dist --non-interactive
+
+  cd:
+    name: EAS Production Build
+    needs: ci
+    if: ${{ ((github.event_name == 'push' && github.ref == 'refs/heads/main') || github.event_name == 'workflow_dispatch') && secrets.EXPO_TOKEN != '' }}
```

actionlint (1.7.11)

[error] context "secrets" is not allowed here. available contexts are "github", "inputs", "needs", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)